infra/postdeployment.tf

```diff
@@ -81,6 +82,6 @@ gcloud beta run jobs execute ${google_cloud_run_v2_job.client.name} --wait --pro
 echo "Warm up API"
 curl ${local.server_url}/api/products/?warmup
 
-# shutdown -h now
+shutdown -h now
```

Comment:
question: How do we know if something goes wrong with the startup. For example, what if the migration job stops working? Does something alert, or is this a place we should consider adding a log-based alert?

Reply:
https://github.com/GoogleCloudPlatform/terraform-dynamic-python-webapp/issues/50